doc/54_Git.md

```diff
@@ -46,15 +46,15 @@ An unstaged change can be reverted by clicking on the `Discard Changes` action n
 location. Double-clicking on a modified file will open it in a diff editor. The read-only editor on
 the left-hand side shows the state from the index. The right-hand side of the editor reflects the
 state of the working tree, and it lets you to further modify the file. New and deleted files do not
-open in a diff editor. The files can be staged (added to the index) by clicking on the `Stage
-Changes` action (`+` sign), the opposite operation being the `Unstage Changes` action (`-` sign).
+open in a diff editor. Files can be staged (added to the index) by clicking on the `Stage
+Changes` action (`+` sign). To unstage a changes, use `Unstage Changes` action (`-` sign).
```

Comment:
To unstage a change...

Reply:
used *unstage changes* in the follow-up. Is that OK?